app/views/posts/index.html.erb

```diff
@@ -1,3 +1,18 @@
-<h1>
-    These are the users posts
-<h1>
\ No newline at end of file
+
+    <div>
+        <img src=<%=@user.photo%>>
+        <div>
+            <p><%=@user.name%></p>
+            <p><%=@user.posts_counter%></p>
+        </div>
+    </div>
+
+    <div>
+        <%@posts.each do |post|%>
+        
+            <%= render partial: "post_card", locals: {post: post, comments: @comments, user: @user}%>

```

Comment:
- [ ] Same here kindly make sure to use `recent_comments` method to render only the last 5 comments in the `posts` index page as required